wait/exec_test.go

```diff
@@ -156,6 +156,27 @@ func TestExecStrategyWaitUntilReady_CustomExitCode(t *testing.T) {
 	}
 }
 
+func TestExecStrategyWaitUntilReadyWithSugaryExitCode(t *testing.T) {
```

Comment:
Not a blocker, but more consistent name with other test functions
```suggestion
func TestExecStrategyWaitUntilReady_ExitCode(t *testing.T) {
```

Reply:
@mdelapenya Updated.